src/main/java/org/primefaces/component/datatable/DataTableBase.java

```diff
@@ -670,6 +670,14 @@ public void setRowEditMode(String rowEditMode) {
         getStateHelper().put(PropertyKeys.rowEditMode, rowEditMode);
     }
 
+    public javax.el.MethodExpression getGlobalFilterFunction() {
```

Comment:
Use simple class name and not with packages please. Good fix though, simple and neat :wink:

Reply:
Sorry about that.. I just did some copy pasting here.

https://github.com/primefaces/primefaces/blob/master/src/main/java/org/primefaces/component/column/ColumnBase.java#L265